src/deepqmc/train.py

```diff
@@ -127,11 +132,20 @@ def train(  # noqa: C901
     if workdir:
         workdir = f'{workdir}/{mode}'
         chkpts = CheckpointStore(workdir, **(chkpts_kwargs or {}))
-        writer = tensorboard.summary.Writer(workdir)
+        global_writer = tensorboard.summary.Writer(f'{workdir}')
+        writers = [
+            tensorboard.summary.Writer(f'{workdir}/{i}') for i in range(len(sampler))
+        ]
```

Comment:
This is fine for now, but this will create a *lot* of plots when we have 1000+ size dataset. We should discuss how best to represent the outputs for very large datasets.

Reply:
I totally agree. There are quite a few questions we'll still have to answer before we can go to very large datasets. I'll try to collect these.